cmd/backup/encrypt_archive.go

```diff
@@ -17,31 +18,91 @@ import (
 // In case no passphrase is given it returns early, leaving the backup file
 // untouched.
 func (s *script) encryptArchive() error {
-	if s.c.GpgPassphrase == "" {
+	if s.c.GpgPassphrase != "" {
```

Comment:
Question: what happens when someone provides both GPG and age configuration? Will the file be double encrypted? While there's nothing technically wrong about this, I wonder if it'd be more user friendly to bail on such a setup? Or maybe print a warning.

Reply:
Good callout, will add a check to ensure configuring age/GPG is `NOT (GPG AND age)`

Reply:
Done: https://github.com/offen/docker-volume-backup/compare/18b7fb576bf27ba159682ace6953074fc03a70a1..58daebfc43280e7ba9406ae6c5681580c4d3288c